test/images.js

```diff
@@ -173,6 +178,91 @@ describe('Images', function() {
     assert(editable.length === 0);
   });
 
+  it('REST: should be able to upload an image with an attachment as an admin', async function() {
+    jar = await login('admin');
+    const formData = new FormData();
+    formData.append('file', fs.createReadStream(path.join(__dirname, '/public/test-image.jpg')));
+
+    // Make an async request to upload the image.
+    const attachment = await apos.http.post('/api/v1/@apostrophecms/attachment/upload', {
+      body: formData,
+      jar
+    });
+
+    image = await apos.http.post('/api/v1/@apostrophecms/image', {
+      body: {
+        title: 'Test Image',
+        attachment
+      },
+      jar
+    });
+    assert(image);
+    assert(image.title === 'Test Image');
+  });
+
+  it('REST: autocrop should have no effect when there are no widget options', async function() {
+    const result = await apos.http.post('/api/v1/@apostrophecms/image/autocrop', {
+      body: {
+        relationship: [ image ],
+        widgetOptions: {}
+      },
+      jar
+    });
+    assert(result.relationship);
+    assert(result.relationship[0]);
+    assert(result.relationship[0].title === 'Test Image');
+    assert(!result.relationship[0]._fields);
+  });
+
+  it('REST: autocrop should work when aspectRatio is less than actual image', async function() {
+    const result = await apos.http.post('/api/v1/@apostrophecms/image/autocrop', {
+      body: {
+        relationship: [ image ],
+        widgetOptions: {
+          aspectRatio: [ 1, 2 ]
+        }
+      },
+      jar
+    });
+    assert(result.relationship);
+    const output = result.relationship[0];
+    assert(output);
+    assert(output.title === 'Test Image');
+    const fields = output._fields;
+    assert(fields);
+    // Useful for visual verification
```

Comment:
Keeping this around, commented out, because it's useful for manual validation that the actual visual result is correct.